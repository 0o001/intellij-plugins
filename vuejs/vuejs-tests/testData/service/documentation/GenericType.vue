<script setup lang="ts">
import {ref} from "vue";

let <error>dummyErrorToSatisfyTestFramework</error>: string = 1;

/**
 * docs have to be removed!
 */
let local = ref(323 as number | string);

<caret>local;
</script>
