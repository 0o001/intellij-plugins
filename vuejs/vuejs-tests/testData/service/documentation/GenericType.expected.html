let local: <a href="psi_element://Ref">Ref</a>&lt;string | number&gt;<br/>[ide]<br/>let local: <a href="psi_element://Ref">Ref</a>&lt;<a href="psi_element://UnwrapRef">UnwrapRef</a>&lt;number&gt;&gt; | <a href="psi_element://Ref">Ref</a>&lt;<a href="psi_element://UnwrapRef">UnwrapRef</a>&lt;string&gt;&gt;